docs/UPDATE.md

```diff
@@ -68,3 +68,15 @@ update:
     - "ignore_me*"
     - "*ignore_me"
 ```
+
+## Version Transform
+
+Some projects create tags than are not compliance with apk format. You can manipulate this with regex on `version-transform` section.
+
```

Comment:
Can you give an example here from the other PR so people don't have to go look around for it?

Reply:
The pr I mentioned in the issue is private, but I create the patch of the changes. What do you think?

Reply:
Sorry, I meant the issue: https://github.com/chainguard-dev/melange/issues/945

Where you could put the relevant details like:

version: "11.0.21.9"

And that needs to be turned into a filename (show the filename example):
`https://github.com/adoptium/temurin11-binaries/releases/download/jdk-${{vars.mangled-name-jdk}}/OpenJDK11U-jdk_x64_linux_hotspot_${{vars.mangled-version-binary}}.tar.gz
`
where you should then what the 11.0.21.9 should turn into for `vars.mangled-name-jdk` as `11.0.21+9` and so forth.

Reply:
Sorry Im not understand :grimacing:.
This details of `11.0.21+9` is not documented [here](https://github.com/chainguard-dev/melange/blob/main/docs/VAR-TRANSFORMS.md?plain=1#L15), with the openjdk version?
One time you modify with `version-transform` you can manuplate the `package.version` with the `var-transforms`

Reply:
I think it's great as is. Thank you so much for doing this!!